index.html

```diff
@@ -0,0 +1 @@
+Hello world
```

Comment:
why this is needed please correct

Reply:
This is to suggest changes i will change it